consensus/src/ordered_broadcast/engine.rs

```diff
@@ -375,14 +375,14 @@ impl<
                     let node = match msg {
                         Ok(node) => node,
                         Err(err) => {
-                            warn!(?err, ?sender, "node decode failed");
+                            debug!(?err, ?sender, "node decode failed");
```

Comment:
Any log that can be triggered via some network message should be no greater than `debug`.